app/gui/src/dashboard/App.tsx

```diff
@@ -532,42 +520,40 @@ function AppRouter(props: AppRouterProps) {
 
   return (
     <FeatureFlagsProvider>
-      <MotionConfig reducedMotion="user" transition={DEFAULT_TRANSITION_OPTIONS}>
```

Comment:
Here we only move the MotionConfig from `<App />` to `<UIProvider />` to reuse it in `Storybook`